test/test_core.py

```diff
@@ -8462,7 +8461,7 @@ def test_wasm2js_fallback(self, args):
     if self.is_wasm2js():
       self.skipTest('redundant to test wasm2js in wasm2js* mode')
 
-    cmd = [EMCC, test_file('small_hello_world.c'), '-sWASM=2'] + args
+    cmd = [EMCC, test_file('small_hello_world.c'), '-sWASM=2', '-sWASM_BIGINT=0'] + args
```

Comment:
This should not be needed I think.

Reply:
I think it is, because if we explicitly set WASM=2 (i.e. require wasm2js) then we throw an error unless WASM_BIGINT is disabled.